ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java

```diff
@@ -243,6 +243,7 @@ public void getUpgradeHiveTableProperties(Context ctx) throws IllegalAccessExcep
         AmsUtil.getNotDeprecatedAndNotInternalStaticFields(TableProperties.class);
     tableProperties.keySet().stream()
         .filter(key -> !key.endsWith("_DEFAULT"))
+        .filter(key -> tableProperties.containsKey(key + "_DEFAULT"))
```

Comment:
If I understand correctly, properties without default values will be ignored. I'm curious to know which specific parameters need to be ignored in this way. Would it be appropriate to consider handling these parameters through HIDDEN_EXPOSED?

Reply:
By my count there are close to 30 props that need to be filtered.